Mimir.E2ETests/ActionPointTests.cs

```diff
@@ -0,0 +1,55 @@
+using Bencodex.Types;
+using HeadlessGQL;
+using Lib9c.Models.Extensions;
+using Lib9c.Models.States;
+using Libplanet.Crypto;
+using Microsoft.Extensions.DependencyInjection;
+using MimirGQL;
+using Nekoyume;
+
+namespace Mimir.E2ETests;
+
+public class ActionPointTests : IClassFixture<GraphQLClientFixture>
+{
+    private readonly IMimirClient mimirClient;
+    private readonly IHeadlessClient headlessClient;
+
+    public ActionPointTests(GraphQLClientFixture fixture)
+    {
+        mimirClient = fixture.ServiceProvider.GetRequiredService<IMimirClient>();
+        headlessClient = fixture.ServiceProvider.GetRequiredService<IHeadlessClient>();
+    }
+
+    [Theory]
+    [InlineData("9bFA9196e93E8186A22757c367b92c74F7B0BeA3")]
+    public async Task CompareAgentDataFromDifferentServices_ShouldMatch(string address)
```

Comment:
Please change the test name

```suggestion
    public async Task CompareActionPointData(string address)
```